include/boost/beast/websocket/impl/read.hpp

```diff
@@ -492,11 +492,12 @@ class stream<NextLayer, deflateSupported>::read_some_op
                             empty_block[4] = { 0x00, 0x00, 0xff, 0xff };
                         zs.next_in = empty_block;
                         zs.avail_in = sizeof(empty_block);
+                        auto const total_out = zs.total_out;
                         impl.inflate(zs, zlib::Flush::sync, ec);
                         if(! ec)
                         {
                             // https://github.com/madler/zlib/issues/280
-                            if(zs.total_out > 0)
+                            if(zs.total_out > total_out)
```

Comment:
I'm not sure this does anything, isn't `zs.total_out` always 0 at line 495? `zs` is always newly constructed on line 465, which sets `total_out` to `0`. Or am I missing something?

Reply:
You're not missing anything, it is always initially 0; confirmed not a bug.